app/src/main/java/com/github/jorgecastillo/kotlinandroid/KotlinArchitectureApp.kt

```diff
@@ -1,16 +1,27 @@
 package com.github.jorgecastillo.kotlinandroid
 
 import android.app.Application
-import kategory.Id
+import com.github.jorgecastillo.architecturecomponentssample.model.error.CharacterError
+import com.github.jorgecastillo.kotlinandroid.di.context.GetHeroesContext
+import com.github.jorgecastillo.kotlinandroid.functional.AsyncResult
+import com.github.jorgecastillo.kotlinandroid.functional.MonadControl
+import kategory.GlobalInstances
+import kategory.InstanceParametrizedType
 
 class KotlinArchitectureApp : Application() {
 
-  override fun onCreate() {
-    super.onCreate()
-    initGlobalInstances()
-  }
+    override fun onCreate() {
+        super.onCreate()
```

Comment:
Don't worry. I'll just try to enable the auto formatter included in `detekt` to be applied automatically by CI so it becomes impossible to upload code not following the required rules.